airflow/www/gunicorn_config.py

```diff
@@ -37,7 +37,5 @@ def on_starting(server):
     from airflow.providers_manager import ProvidersManager
 
     providers_manager = ProvidersManager()
-    # Load providers configuration before forking workers
```

Comment:
Why removing this one? I think I've added that one as result of a subtle problem that defaults  were not loaded by gunicorn workers and providers need to be initialized before getting connection form widgets.

Reply:
My thinking is that moving it to Appbuilder initialization will do the same thing

Reply:
I will return it but I feel it would be about repeating provider configuration loading